modules/ting_das/ting_das.info

```diff
@@ -0,0 +1,10 @@
+name = Ting digital article service
+description = Use Digital Articel Service from the Royal Danish Library to order copies of articles.
```

Comment:
Nitpicking (but it is in the module description after all): Articel = Article.

Reply:
Fixed